depends/hosts/netbsd.mk

```diff
@@ -0,0 +1,31 @@
+netbsd_CFLAGS=-pipe
+netbsd_CFLAGS_CXXFLAGS=$(netbsd_CFLAGS)
```

Comment:
Should be `netbsd_CXXFLAGS`, no?

Looks like maybe the same search/replace problem with the freebsd host too?